src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -96,7 +96,7 @@ const Header = ({item, image, name, time, showBackButton, follow, setFollow}) =>
         <Pressable onPress={onHeaderClicked} style={styles.leftRowContainer}>
           {showBackButton && (
             <TouchableOpacity onPress={onNavigationBack} style={styles.backbutton}>
-              <MemoIc_arrow_back width={18} height={18} fill={COLORS.white2} />
+              <MemoIc_arrow_back width={18} height={18} fill={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the back arrow icon has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the visibility of the icon against its background, especially in different themes or modes (like dark mode) if applicable.
```diff
- <MemoIc_arrow_back width={18} height={18} fill={COLORS.white2} />
+ <MemoIc_arrow_back width={18} height={18} fill={COLORS.white} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->